app/plugin_deadlock_test.go

```diff
@@ -4,6 +4,7 @@
 package app
 
 import (
+	"github.com/stretchr/testify/require"
```

Comment:
Indentation here seems off, or is it just GitHub?

Reply:
@lieut-data probably GitHub, `make gofmt` makes no changes on my side

Reply:
@lieut-data it's `Refined Github` that's causing the indentation issue. I'm seeing the same. Probably an issue on their end.